src/Microsoft.AspNet.SignalR.SqlServer/DbOperation.cs

```diff
@@ -180,43 +168,28 @@ private void LoggerCommand(DbCommand command)
         [SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times", Justification = "Disposed in async Finally block"),
          SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope", Justification = "Disposed in async Finally block")]
 #if ASPNET50
-        private void Execute<T>(Func<IDbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
+        private async void Execute<T>(Func<IDbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
 #else
-        private void Execute<T>(Func<DbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
+        private async void Execute<T>(Func<DbCommand, Task<T>> commandFunc, TaskCompletionSource<T> tcs)
```

Comment:
`async void` methods are for events. If the methods throws you will have no way of knowing it. If you want to do fire and forget I think you can use `Task.Run()` which will also remove the warning.